### PR TITLE
Fix #8840: Selenium test framework: TimeoutException when downloading a (text) file

### LINKED ISSUE #8840: Selenium test framework: TimeoutException when downloading a (text) file
### Describe the bug

Running a Selenium test on Firefox featuring the download of a (text) file fails with TimeoutException.

Remark: I have also seen UnsupportedOperationException during file download (maybe with HTMLUnitDriver which is not yet supported by PF Selenium test framework) when the onload.js sees an "empty" http response.

### Reproducer

Will provide a pull request with an appropriate integration test producing the error.

### Expected behavior

Selenium test runs fine without TimeoutException.

### PrimeFaces edition

Community

### PrimeFaces version

11

### Theme

Any

### JSF implementation

MyFaces

### JSF version

2.2, 2.3

### Browser(s)

Firefox 100, maybe all browsers

### PR DESCRIPTION
**Pull request by nettozahler**
Pull request for issue #8840

### CODE REVIEW

**Comment by jepsar** [comment]
@nettozahler this PR is empty, did you push the correct branch?

**Comment by nettozahler** [comment]
Let me try again later...
